config/routes.rb

```diff
@@ -4,7 +4,8 @@
 
   namespace 'api' do
     namespace 'v1', :constraints => {format: 'json'} do
-      resources :registrations, :controller => "users", :only=> [:create]
+      resources :registrations, :controller => "users", :only => [:create]
```

Comment:
I wanted to create a different route for user registration like api/v1/registrations independent of users.